### ISSUE TITLE
lost original type of element in Set when invoke toArray()

### ISSUE THREAD

**Issue by edisonX-sudo** [pvorb/node-set#6]
`
new Set([1,2,3]).toArray() // suppose: [1,2,3] actual:['1','2','3']
`

**Comment by edisonX-sudo** [comment]
fill this.set with the original value instead of `true` may solve this problem

**Comment by paulharris** [comment]
No, the design of this set is to convert all the entered items into strings ie via builtin toString, and add to the set as keys (with value=true).

So what you see is by design, so anything can be used as a key (not just numbers).
